packages/chart/src/index.ts

```diff
@@ -1,3 +1,7 @@
+// Used to show many WebGL plots on the same page
+// https://github.com/plotly/plotly.js/?tab=readme-ov-file#need-to-have-several-webgl-graphs-on-a-page
+import './virtual-webgl';
```

Comment:
Just curious if you tried the webgl2 version. I know the plotly readme says webgl1 specifically, but curious if webgl2 would work. I don't care to find out why it won't if it doesn't, just curious if you tried

Reply:
WebGL2 did not work and threw errors.